app/controllers/partners_controller.rb

```diff
@@ -129,18 +129,6 @@ def invite
     end
   end
 
-  def invite_partner_user
```

Comment:
This is still referenced from config/routes.rb, but nothing submits to it, so not an immediate blocker